packages/application-components/src/components/project-stamp/project-stamp.tsx

```diff
@@ -0,0 +1,63 @@
+import { css } from '@emotion/react';
+import { useIntl } from 'react-intl';
+import { DotIcon } from '@commercetools-uikit/icons';
+import Stamp, { TTone } from '@commercetools-uikit/stamp';
+import messages from './messages';
+
+type TProjectStampProps = {
+  isProductionProject?: boolean | undefined;
+  isSuspended?: boolean | undefined;
+  isExpired?: boolean | undefined;
+  willExpire?: boolean | undefined;
+  daysLeft?: number | undefined;
+};
+
+function ProjectStamp(props: TProjectStampProps) {
+  const intl = useIntl();
+
+  const renderStamp = (tone: TTone, label: string, icon?: JSX.Element) => (
+    <Stamp tone={tone} isCondensed={true} label={label} icon={icon} />
+  );
+
+  return (
+    <div
+      css={css`
+        > * {
+          width: max-content;
+        }
+      `}
+    >
```

Comment:
Can we provide and expose individual components instead of a big one?

Otherwise it makes it harder to integrate in different scenarios.

Thanks

Reply:
@emmenko was something like this what you had in mind? [Commit](https://github.com/commercetools/merchant-center-application-kit/commit/23a0f2338681db1f61ea3f4fe40f01dc12d30506)

Reply:
Yeah kind of.

For the "expire" case we should have only one component though. And I would suggest to also include the "not initialized" case.

So to recap we can expect to have:

```
ProjectProductionStamp
ProjectExpiredStamp
ProjectSuspendedStamp
ProjectNotInitializedStamp
`
``